components/gitpod-db/src/typeorm/workspace-cluster-db-impl.ts

```diff
@@ -78,6 +78,9 @@ export class WorkspaceClusterDBImpl implements WorkspaceClusterDB {
         if (predicate.url !== undefined) {
             qb = qb.andWhere("wsc.url = :url", predicate);
         }
+        if (predicate.region !== undefined) {
```

Comment:
should this also check for the unknown region `""`?

Reply:
Good spot. Currently it would make the filter effectively required, and it would use "" as the default filter value.